src/dmd/dtemplate.d

```diff
@@ -1351,8 +1351,37 @@ extern (C++) final class TemplateDeclaration : ScopeDsymbol
             {
                 Parameter fparam = fparameters[parami];
 
+                Type prmtype = fparam.type;
+                //printf("prmtype: %s\n", prmtype.toChars());
+                while(prmtype.ty == Tinstance)
+                {
+                    TypeInstance prmti = cast(TypeInstance) prmtype;
+                    //printf("prmti: %s\n", prmti.toChars());
+                    if(prmti.tempinst.findTempDecl(sc, null))
+                    {
+                        TemplateDeclaration td = prmti.tempinst.tempdecl.isTemplateDeclaration();
+                        if(td)
+                        {
+                            //printf("td: %s\n", td.toChars());
+                            if(td.onemember)
+                            {
+                                AliasDeclaration ad = td.onemember.isAliasDeclaration();
+                                if(ad)
```

Comment:
same here, `if (AliasDeclaration ad = td.onemember.isAliasDeclaration())`

Reply:
Got it, for both, I just thought it would be too long for a condition. Also yes, bad decision to go with the master.